addons/project/static/src/js/tours/project_task_history.js

```diff
@@ -0,0 +1,123 @@
+/** @odoo-module */
+
+/**
+ * Project Task history tour.
+ * Features tested:
+ * - Create / edit a task description and ensure revisions are created on write
+ * - Open the history dialog and check that the revisions are correctly shown
+ * - Select a revision and check that the content / comparison are correct
+ * - Click the restore button and check that the content is correctly restored
+ */
+
+import { registry } from "@web/core/registry";
+import { stepUtils } from "@web_tour/tour_service/tour_utils";
+import { openCommandBar } from "@web_editor/js/editor/odoo-editor/src/utils/utils";
+
+const baseDescriptionContent = "Test project task history version";
+const descriptionField = "div.note-editable.odoo-editor-editable.odoo-editor-qweb p";
+function changeDescriptionContentAndSave(newContent) {
+    const newText = `${baseDescriptionContent} ${newContent}`;
+    return [ {
+        trigger: descriptionField,
+        run: async function(actions) {
+            const textTriggerElement = this.$anchor.find(descriptionField).get(0);
+            actions.text(newText, textTriggerElement);
+            await new Promise((r) => setTimeout(r, 300));
+        },
+    }, {
+        trigger: "button.o_form_button_save",
+    }];
+}
+
+registry.category("web_tour.tours").add("project_task_history_tour", {
+    test: true,
+    url: "/web",
+    steps: () => [stepUtils.showAppsMenuItem(), {
+        content: "Open the project app",
```

Comment:
It should be translated, I will fix it 
```suggestion
        import { markup } from "@odoo/owl";
        ... 
        content: markup(_t("Open the project app",
```